tests/unit/test_saml.py

```diff
@@ -42,22 +51,17 @@ def test_saml_with_invalid_metadata(urlopen_mock):
 
 
 @patch.object(urllib.request, "urlopen", side_effect=urllib.error.URLError("Error"))
-def test_saml_with_invalid_url(urlopen_mock):
+def test_saml_with_invalid_url(_):
```

Comment:
May I ask if the `_` is required as a placeholder for the object patch?

Reply:
Yes, it is required